service/assessment/assessment.go

```diff
@@ -390,14 +390,16 @@ func (svc *Service) handleEvidence(ev *evidence.Evidence, resourceId string) (er
 		result := &assessment.AssessmentResult{
 			Id:             uuid.NewString(),
 			Timestamp:      timestamppb.Now(),
-			CloudServiceId: ev.CloudServiceId,
+			CloudServiceId: ev.GetCloudServiceId(),
 			MetricId:       metricID,
 			MetricConfiguration: &assessment.MetricConfiguration{
-				TargetValue: convertedTargetValue,
-				Operator:    data.Operator,
+				TargetValue:    convertedTargetValue,
```

Comment:
UpdatedAt and is_default missing from `data`

Reply:
Data has only these fields, I think that was the reason why I used the GetMetricConfiguration()

`type Result struct {
	Applicable  bool
	Compliant   bool
	TargetValue interface{} `mapstructure:"target_value"`
	Operator    string
	MetricId    string
}`

Reply:
You can add the missing fields to the `Result` struct and you need to add the missing lines here to fill the result struct:
https://github.com/clouditor/clouditor/blob/338742c6d17779385e2829a917a39c87533c54d5/policies/rego.go#L254-L255